cli/lib/tasks/verify.js

```diff
@@ -65,7 +65,17 @@ const runSmokeTest = (binaryDir) => {
 
     debug('smoke test command:', smokeTestCommand)
 
-    return Promise.resolve(util.exec(cypressExecPath, args))
+    function smokeTestExec () {
+      let child = util.exec(cypressExecPath, args)
+
+      if (child.stderr) {
```

Comment:
Looking at https://github.com/sindresorhus/execa/tree/v0.10.0, no idea why sometimes `stderr` isn't there, but the unit tests fail without this guard 🤷‍♂️ .